packages/wpcom.js/src/index.js

```diff
@@ -166,19 +154,18 @@ WPCOM.prototype.freshlyPressed = function( query, fn ) {
 	return this.req.get( '/freshly-pressed', query, fn );
 };
 
-/**
- * Expose send-request
- *
- * @TODO: use `this.req` instead of this method
- */
+// Expose send-request
+// TODO: use `this.req` instead of this method
 WPCOM.prototype.sendRequest = function( params, query, body, fn ) {
-	var msg = 'WARN! Don use `sendRequest() anymore. Use `this.req` method.';
+	const msg = 'WARN! Don use `sendRequest() anymore. Use `this.req` method.';
```

Comment:
This function also seems to be a legacy stuff that we could remove when doing a major release. (in a different PR for sure)